server/src/main/java/io/crate/metadata/DanglingArtifactsService.java

```diff
@@ -70,12 +62,11 @@ protected void doClose() {
     public void clusterChanged(ClusterChangedEvent event) {
         if (LOGGER.isInfoEnabled() && event.isNewCluster()) {
             for (ObjectCursor<String> key : event.state().metadata().indices().keys()) {
-                for (Pattern pattern : danglingPatterns) {
```

Comment:
Is this supposed to be part of this PR?

Reply:
Good point, I'll extract it into a separate PR. My first iteration had looked a bit different and required changes here